src/runtime/extra_prints.c

```diff
@@ -240,7 +240,23 @@ static void print_timestamp(string b, timestamp t)
 static void format_timestamp(buffer dest, struct formatter_state *s, vlist *a)
 {
     timestamp t = varg(*a, timestamp);
-    print_timestamp(dest, t);
+    print_decimal(dest, t, 3);
```

Comment:
This changes the resolution of printed timestamp values from microseconds to milliseconds. Is that intentional?

Reply:
No, that should still be six decimal digits. I've changed it back, thanks!